dll/win32/shell32/wine/classes.c

```diff
@@ -334,6 +334,108 @@ BOOL HCR_GetIconA(LPCSTR szClass, LPSTR szDest, LPCSTR szName, DWORD len, int* p
 	return ret;
 }
 
+#ifdef __REACTOS__
+BOOL HCU_GetIconW(LPCWSTR szClass, LPWSTR szDest, LPCWSTR szName, DWORD len, int* picon_idx)
+{
+    HKEY hkey;
+    WCHAR sTemp[MAX_PATH];
+    BOOL ret = FALSE;
+
+    TRACE("%s\n", debugstr_w(szClass));
+
+    swprintf(sTemp, L"%s\\DefaultIcon", szClass);
```

Comment:
Could you use string-safe functions here?
```suggestion
    StringCchPrintfW(sTemp, _countof(sTemp), L"%s\\DefaultIcon", szClass);
```